lang/shared/src/main/scala/com/wavesplatform/lang/v1/repl/node/http/response/model/Transaction.scala

```diff
@@ -92,7 +92,7 @@ object Transaction {
       version.toByte,
       senderPublicKey,
       proofs,
-      bodyBytes,
+      ByteString(),
```

Comment:
how to get real bodyBytes?

Reply:
https://jira.wavesplatform.com/browse/SC-631